src/ape/api/accounts.py

```diff
@@ -72,6 +72,13 @@ def alias(self) -> Optional[str]:
         """
         return None
 
+    @property
+    def public_key(self) -> Optional[HexBytes]:
+        """
+        The public key for the account.
+        """
```

Comment:
```suggestion

        ```{notice}
        Account might not have this property if feature is unsupported or inaccessible.
        ```
        """
```